tests/hazmat/primitives/test_aead.py

```diff
@@ -495,10 +495,20 @@ def test_data_too_large(self):
             aesocb3.encrypt(nonce, b"", FakeData())
 
     def test_vectors(self, backend, subtests):
-        vectors = load_vectors_from_file(
-            os.path.join("ciphers", "AES", "OCB3", "rfc7253.txt"),
-            load_nist_vectors,
-        )
+        vectors = []
+        for f in (
+            "rfc7253.txt",
+            "test-vector-1-nonce104.txt",
+            "test-vector-1-nonce112.txt",
+            "test-vector-1-nonce120.txt",
+        ):
```

Comment:
```suggestion
        for f in [
            "rfc7253.txt",
            "test-vector-1-nonce104.txt",
            "test-vector-1-nonce112.txt",
            "test-vector-1-nonce120.txt",
        ]:
```

nit: tuples are heterogenous records, list are homogeneous sequences